spec/rails_helper.rb

```diff
@@ -9,6 +9,7 @@
 require File.join(File.dirname(__FILE__), 'support/valid_attribute')
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
 require File.join(File.dirname(__FILE__), 'support/database_cleaner')
+require File.join(File.dirname(__FILE__), 'helpers/fill_in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.